go.mod

```diff
@@ -175,6 +175,7 @@ require (
 	github.com/onflow/atree v0.6.1-0.20230711151834-86040b30171f // indirect
 	github.com/onflow/cadence-tools/lint v1.0.0-M1 // indirect
 	github.com/onflow/crypto v0.25.0 // indirect
+	github.com/onflow/flow-cli/flowkit v1.11.1-0.20240130210637-a22f7c578d37 // indirect
```

Comment:
this will continue showing up until we move over `languageserver` but thats not super important